<!DOCTYPE html><html xml:lang="en" lang="en" xmlns="http://www.w3c.org/1999/xhtml">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <title>LAB17</title>
    <meta name="title" content="NHANES 1999-2000: Cryptosporidum &amp; Toxoplasma Data Documentation, Codebook, and Frequencies" />
    <link rel="stylesheet" href="https://wwwn.cdc.gov/nchs/nhanes/css/documentation.css" />
  </head>
  <body class="nhanes-doc">
    <div id="toc">
      <h3>Table of Contents</h3>
      <div id="toc-scroll">
        <ul>
          <li>
            <a href="#Component_Description">Component Description</a>
          </li>
          <li>
            <a href="#Eligible_Sample">Eligible Sample</a>
          </li>
          <li>
            <a href="#Description_of_Laboratory_Methodology">Description of Laboratory Methodology</a>
          </li>
          <li>
            <a href="#Data_Processing_and_Editing">Data Processing and Editing</a>
          </li>
          <li>
            <a href="#Laboratory_Quality_Assurance_and_Monitoring">Laboratory Quality Assurance and Monitoring</a>
          </li>
          <li>
            <a href="#Analytic_Notes">Analytic Notes</a>
          </li>
          <li>
            <a href="#Codebook">Codebook</a>
            <ul id="CodebookLinks">
              <li>
                <a href="#SEQN">SEQN - Respondent sequence number</a>
              </li>
              <li>
                <a href="#LBDC1">LBDC1 - Cryptosporidium (17kDA)</a>
              </li>
              <li>
                <a href="#LBDC2">LBDC2 - Cryptosporidium (27kDA)</a>
              </li>
              <li>
                <a href="#LBXTO1">LBXTO1 - Toxoplasma (IgG)</a>
              </li>
              <li>
                <a href="#LBXTO2">LBXTO2 - Toxoplasma (IgM)</a>
              </li>
              <li>
                <a href="#LBXTO3">LBXTO3 - Toxoplasma (Dye)</a>
              </li>
              <li>
                <a href="#LBXTO4">LBXTO4 - Toxoplasma Differential Agglutination</a>
              </li>
              <li>
                <a href="#LBXTO4IN">LBXTO4IN - Toxoplasma Agglutin interpretation</a>
              </li>
              <li>
                <a href="#LBXTO5">LBXTO5 - Toxoplasma (Avidity) IgG</a>
              </li>
              <li>
                <a href="#LBXTO5IN">LBXTO5IN - Toxoplasma (Avidity IgG) interpretation</a>
              </li>
            </ul>
          </li>
        </ul>
      </div>
    </div>
    <div id="wrapper">
      <div id="PageHeader">
        <h1>National Health and Nutrition Examination Survey</h1>
        <h2>1999-2000 Data Documentation, Codebook, and Frequencies</h2>
        <h3>Cryptosporidum &amp; Toxoplasma (LAB17)</h3>
        <h4> Data File: LAB17.xpt</h4>
        <h5 class="left">First Published: June 2002</h5>
        <h5 class="right">Last Revised: NA</h5>
      </div>
      <div id="Sections">
        <h2 id="Component_Description">Component Description</h2><p><strong>1.1 Cryptosporidium</strong> </p>
<p>Cryptosporidium is an important cause of outbreaks of waterborne disease in the United States. In 1993, two outbreaks of cryptosporidiosis occurred in large metropolitan areas (Milwaukee and Las Vegas/Clark County) and were associated with deaths among immuno-compromised persons. While these waterborne outbreaks are increasingly being recognized, it is unclear how much endemic waterborne cryptosporidiosis occurs due to low-level contamination of drinking water. To provide an estimate of exposure to this organism, the prevalence of antibodies to cryptosporidium will be measured in NHANES participants aged 6-49 years. This estimate will support a CDC/EPA response to a Congressional mandate to evaluate the burden of waterborne disease in the United States. </p>
<p>According to a 1999 CDC report, toxoplasmosis is the third leading cause of death due to food borne infections and leads to an estimated 400 to 6,000 cases of congenital infection in the United States. The accuracy of these estimates, however, is unknown. Evidence from Europe suggests that the incidence of toxoplasmosis has dropped substantially in the last decade; data from the U.S. is less clear cut but also suggests a decline. Serologic tests are available to determine who has become infected with toxoplasmosis. Toxoplasma-specific IgG antibodies are detectable 1-3 weeks after infection and remain detectable for the life of the individual. Toxoplasma-specific IgM antibodies are also detectable 1-3 weeks after infection but generally decline to nil by one year after infection. The presence of both IgG and IgM is evidence for infection within the last year. The presence of IgG antibody without IgM is considered indicative of past infection. Serum tests for both IgG and IgM antibody to toxoplasmosis will be added to the NHANES 2001 laboratory protocol to obtain the most accurate information available about the prevalence of toxoplasmosis in the U.S. and determine if the rates are changing over time. Toxoplasma IgG antibody was measured in NHANES III with an overall prevalence of 22.5%.</p>
<p><strong>1.2 Laboratory</strong> </p>
<p>Blood and urine specimens are collected on participants aged one year and older at the mobile examination centers (MECs). Hematological profiles are completed for all participants, and specified laboratory tests are performed upon each specimen based on the participant's age at time of interview and sex.  </p>
<p>The laboratory component of NHANES includes the collection, processing, storage, and shipping of blood, urine, and other biological and environmental specimens. The blood collection procedure consists of administering a questionnaire to screen for conditions that excludes the participants from the blood draw and determines fasting status, a blood draw, and collecting specimens for special studies. The urine collection procedure consists of urine collection, urine processing, and pregnancy testing. The Coulter® MAXM performs a complete blood count on blood specimens to provide a study of blood cells and coagulation for all participants. </p>
<h2 id="Eligible_Sample">Eligible Sample</h2><p>Participants aged 6 to 49 years who do not meet any of the exclusion criteria&#160;<br />
&#160;<br />
<strong>Exclusion Criteria</strong> </p>
<ul>
    <li>Hemophiliacs </li>
    <li>Participants who received chemotherapy within last 4 weeks </li>
    <li>The presence of the following on both arms: rashes, gauze dressings, casts, edema, paralysis, tubes, open sores or wounds, withered arms or limbs missing, damaged, sclerosed or occluded veins, allergies to cleansing reagents, burned or scarred tissue, shunt or IV. </li>
</ul>
<h2 id="Description_of_Laboratory_Methodology">Description of Laboratory Methodology</h2><p><strong>Data Collection Methods</strong> </p>
<p>The MEC phlebotomist draws blood from the participant&#8217;s arm. In the laboratory the blood is processed, stored and shipped to the Division of Parasitic Diseases, National Center for Infectious Diseases, Centers for Disease Control and Prevention for analysis.</p>
<p><strong>Examination Protocol</strong> </p>
<p>Detailed specimen collection and processing instructions are discussed in the NHANES Laboratory/Medical Technologists Procedures Manual (LPM). Vials were stored under appropriate frozen (-20 degrees Centigrade) conditions until they were shipped to Division of Parasitic Diseases, National Center for Infectious Diseases, Centers for Disease Control and Prevention for testing. The analytical methods are described in the Analytic methodology section. </p>
<p><strong>Survey Staff</strong> </p>
<p>The NHANES 1999-2000 laboratory staff consists of medical technologists and phlebotomists. The medical technologists hold baccalaureates in medical technology. The American Society for Clinical Pathologists or a similar organization certifies the medical technologists and the phlebotomists. All laboratory staff completes comprehensive training in standardized laboratory procedures before they begin working in the MEC. The MEC phlebotomists complete comprehensive training in pediatric phlebotomy techniques, including instruction by a pediatric nurse practitioner. </p>
<p><strong>Data Collection Forms</strong> </p>
<p>Detailed specimen collection and processing instructions are discussed in the NHANES Laboratory/Medical Technologists Procedures Manual (LPM). Each chapter in the LPM specifies the procedure to be used for preparation of the participant, specimen collection, labeling, processing, and preservation, and conditions for specimen transport that are appropriate for that method. <br />
<br />
<strong>Analytic Methodology </strong></p>
<p><strong>Cryptosporidium</strong></p>
<p><strong>Recombinant protein expression</strong> <br />
The following two deoxyoligonucleotides were designed for the directional cloning of the C. parvum 27-kDa antigen (GenBank accession number U34390) into the BamHI and EcoRI restriction enzyme sites of the pGEX 4T-2 expression vector (Pharmacia Biotech, Uppsala, Sweden): Cp23 5'-primer (5'-CGC GGA TCC ATG GGT TGT TCA TCA TCA AAG-3') and Cp23 3'-primer (5'-GCG GAA TTC ATT AGG CAT CAG CTG GCT TG-3'). The 27-kDa antigen coding sequence was amplified from 260 ng of genomic DNA by using 100 &#956;M concentrations of Cp23-5' and Cp23-3' and AmpliTaq DNA polymerase as directed by the manufacturer (Perkin-Elmer Cetus, Norwalk, Conn.). The following amplification protocol was used: 30 cycles of 94<sup>o</sup>C for 1 min, 55<sup>o</sup>C for 2 min, and 72<sup>o</sup>C for 3 min followed by 1 cycle of 72<sup>o</sup>C for 15 min. Plasmid containing insert was transformed into Escherichia coli strain HB101 cells (Life Technologies, Frederick, Md.). The sequence of the resulting clone was confirmed by automated DNA sequencing. A recombinant C. parvum 27-kDa antigen/Schistosoma japonicum glutathione-S-transferase (GST) fusion protein was purified from isopropyl-b-D-thiogalactopyranoside (IPTG) induced cell cultures using glutathione Sepharose 4B as directed by the manufacturer (GST Bulk Purification Module, Pharmacia Biotech). The C. parvum protein with an additional GlySer dipeptide at the amino terminus was released by overnight cleavage with thrombin at room temperature and then separated from uncleaved fusion protein and the GST cleavage product by passage over glutathione Sepharose 4B resin. Protein purity was monitored by both SDS polyacrylamide gel electrophoresis and Western blotting with a monoclonal antibody against the native 27-kDa antigen (C6B6) and with serum samples from infected humans. <br />
<br />
<strong>Enzyme-linked immunosorbent assay</strong> <br />
Antigens diluted in 0.1 M NaHCO<sub>3</sub> buffer (pH 9.6) were used to sensitize 96-well plates overnight at 4oC (Immulon 2 flat-bottom microtiter immunoassay plates, Dynatech Industries, Inc., McLean, Va.). Each well contained 50 &#956;l of either the recombinant 27-kDa antigen (0.2&#956;g/ml) (BCA protein microassay, Pierce Biotech Company, Rockford, Ill.). The plates were washed in 0.05% Tween-20 PBS and blocked with 0.3% Tween-20 PBS for 1 hour at 4oC. After a series of three washes (subsequent washes were all with 0.05% Tween-20 PBS), 50 &#956;l aliquots of serum diluted 1:50 with wash buffer were added to each well. All serum samples were tested in duplicate. A two-fold serial dilution (1:50 to 1:6400) of a strong positive control was used to generate a standard curve on each individual plate. Two buffer blanks, three confirmed positive sera, and a battery of four serum samples known by Western blot assay to be negative for C. parvum antibodies were also included on each plate. Plates were incubated 2 hours at room temperature and then washed four times with wash buffer. A biotinylated mouse monoclonal antibody against human IgG (clone HP6017, Zymed Laboratories, 50 &#956;l of a 1:1000 dilution in wash buffer) was added to each well and incubated for 1 hour at room temperature. Following four washes, the wells were filled with alkaline phosphatase-labeled streptavidin (Life Technologies, 50 &#956;l of a 1:500 dilution in wash buffer) and incubated an additional hour at room temperature. After four washes (the final wash for 10 minutes at room temperature), p-nitrophenylphosphate substrate was added in 3 mM MgCl2 and 10% diethanolamine at pH 10, and the color was allowed to develop until the 1:50 positive control wells had reached an absorbance of about 1.5 at 405 nm. Absorbances were measured using a Molecular Devices UVmax kinetic microplate reader. Antibody levels of the unknown samples were assigned a unit value based on the 8-point positive control standard curve with a four parameter curve fit. The 1:50 dilution of the positive control was arbitrarily assigned a value of 6400 units. Unknown samples with absorbance values above the standard curve were diluted further and reassayed. Arbitrary unit values were expressed per microliter of serum. </p>
<p>&nbsp;</p>
<h2 id="Data_Processing_and_Editing">Data Processing and Editing</h2><p>Automated data collection procedures for the survey were introduced in NHANES 1999-2000. In the mobile examination centers (MECs) and analytical laboratories, data for the laboratory component is recorded directly onto a computerized data collection form. The system is centrally integrated and it allows for ongoing monitoring of much of the data. While the complete blood count and pregnancy analyses are performed in the MEC laboratory, most analyses are conducted elsewhere by approximately 21 laboratories across the United States. </p>
<p>Guidelines are developed that provided standards for naming variables, filling missing values, and handling missing records. NCHS staff, assisted by contract staff, develops data editing specifications that check data sets for valid codes, ranges, and skip pattern consistencies and examine the consistency of values between interrelated variables. Comments are reviewed and recoded. NCHS staff verifies extremely high and low values whenever possible, and numerous consistency checks are performed. Nonetheless, users should examine the range and frequency of values before analyzing data. </p>
<p>For laboratory tests with a lower detection limit, results below the lower detection limit are replaced with a value equal to the detection limit divided by the square root of two. This value is created to help the user distinguish a nondetectable laboratory test result from a measured laboratory test result. </p>
<p>The data editing specifications are as follows: </p>
<ul>
    <li>Age and gender checks </li>
    <li>Total number of observations complete for each field</li>
    <li>No field overlap, truncated values, or weird results</li>
    <li>Direct data entry (DDE) errors</li>
    <li>Abnormal results confirmed by lab </li>
    <li>Test algorithm performed </li>
    <li>Checked comment codes to resolve missing results and missing records </li>
    <li>All missing results and missing MEC-examined records are accounted </li>
    <li>Duplicate records are verified and deleted </li>
    <li>Apply the SI conversion </li>
    <li>Apply the below detection limit formula. </li>
</ul>
<p>&nbsp;</p>
<h2 id="Laboratory_Quality_Assurance_and_Monitoring">Laboratory Quality Assurance and Monitoring</h2><p><strong>MEC</strong></p>
<p>Laboratory team performance is monitored using several techniques. NCHS and contract consultants use a structured quality assurance evaluation during unscheduled visits to evaluate both the quality of the laboratory work and the quality-control procedures. Each laboratory staff person is observed for equipment operation, specimen collection and preparation, and testing procedures. Constructive feedback is given to each staff. Formal retraining sessions are conducted annually to ensure that required skill levels are maintained. </p>
<p>The NHANES quality control and quality assurance protocols meet the 1988 Clinical Laboratory Improvement Amendments mandates. Detailed quality control and quality assurance instructions are discussed in the NHANES Laboratory/Medical Technologists Procedures Manual (LPM).</p>
<p><strong>Analytical laboratories</strong> </p>
<p>NHANES uses several methods to monitor the quality of the analyses performed by the contract laboratories. In the MEC these methods include performing second examinations on previously examined participants and blind split samples collected on "dry run" sessions. In addition, contract laboratories randomly perform repeat testing on 2.0 percent of all specimens. </p>
<p>NCHS developed and distributed a quality control protocol for all the Contract laboratories outlining the Westgaard rules used when running NHANES specimens. Progress reports containing any problems encountered during shipping or receipt of specimens, summary statistics for each control pool, QC graphs, instrument calibration, reagents, and any special considerations are submitted to NCHS and Westat quarterly. The reports are reviewed for trends or shifts in the data. The laboratories are required to explain any identified areas of concern. NCHS and Westat are currently reviewing these reports. </p>
<h2 id="Analytic_Notes">Analytic Notes</h2><p>LBXTO1 is required for ALL. </p>
<p>LBXTO2 is only required if LBXTO1 &gt;=7.0. </p>
<p>LBXTO3 is only required if is LBXTO2 &gt;=2.0. </p>
<p>LBXTO4 and LBXTO4IN are only required if LBXTO3 is &gt;=16. </p>
<p>LBXTO5 and LBXTO5IN is only required if LBXTO4 is &gt; 25/ and &gt;50. </p>
<p><strong>Special Notes on Using the Dataset</strong> </p>
<p>The analysis of NHANES 1999-2000 laboratory data must be conducted with the key survey design and basic demographic variables. The NHANES 1999-2000 Household Questionnaire Data Files contain demographic data, health indicators, and other related information collected during household interviews. They also contain all survey design variables and sample weights for these age groups. The phlebotomy file includes auxiliary information such the conditions precluding venipuncture. The household questionnaire and phlebotomy files may be linked to the laboratory data file using the unique survey participant identifier SEQN. </p>
</div>
      <div id="Codebook">
        <h2>Codebook and Frequencies</h2>
        <div class="pagebreak">
          <h3 class="vartitle" id="SEQN">SEQN - Respondent sequence number</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">SEQN</dd>
            <dt>SAS Label: </dt>
            <dd>Respondent sequence number</dd>
            <dt>English Text: </dt>
            <dd class="info">Respondent sequence number.</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBDC1">LBDC1 - Cryptosporidium (17kDA)</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBDC1</dd>
            <dt>SAS Label: </dt>
            <dd>Cryptosporidium (17kDA)</dd>
            <dt>English Text: </dt>
            <dd class="info">Cryptosporidium (17kDA)
</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
            <dt>Hard Edits: </dt>
            <dd class="info">0 to 999999</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">1</td>
                <td class="values">Positive</td>
                <td class="values" align="right">1199</td>
                <td class="values" align="right">1199</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">2</td>
                <td class="values">Negative</td>
                <td class="values" align="right">3891</td>
                <td class="values" align="right">5090</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">573</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBDC2">LBDC2 - Cryptosporidium (27kDA)</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBDC2</dd>
            <dt>SAS Label: </dt>
            <dd>Cryptosporidium (27kDA)</dd>
            <dt>English Text: </dt>
            <dd class="info">Cryptosporidium (27kDA)
</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
            <dt>Hard Edits: </dt>
            <dd class="info">0 to 999999</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">1</td>
                <td class="values">Positive</td>
                <td class="values" align="right">3458</td>
                <td class="values" align="right">3458</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">2</td>
                <td class="values">Negative</td>
                <td class="values" align="right">1632</td>
                <td class="values" align="right">5090</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">573</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO1">LBXTO1 - Toxoplasma (IgG)</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO1</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma (IgG)</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma (IgG)</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">0 to 240</td>
                <td class="values">Range of Values</td>
                <td class="values" align="right">5089</td>
                <td class="values" align="right">5089</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">574</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO2">LBXTO2 - Toxoplasma (IgM)</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO2</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma (IgM)</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma (IgM)</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">0 to 9.5</td>
                <td class="values">Range of Values</td>
                <td class="values" align="right">667</td>
                <td class="values" align="right">667</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">4996</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO3">LBXTO3 - Toxoplasma (Dye)</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO3</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma (Dye)</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma (Dye) Positive =&gt; 1:8</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">8 to 16000</td>
                <td class="values">Range of Values</td>
                <td class="values" align="right">106</td>
                <td class="values" align="right">106</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">5557</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO4">LBXTO4 - Toxoplasma Differential Agglutination</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO4</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma Differential Agglutination</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma Differential Agglutination</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">Toxoplasma Differential Agglutination</td>
                <td class="values">Value was recorded</td>
                <td class="values" align="right">74</td>
                <td class="values" align="right">74</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">&lt; blank &gt;</td>
                <td class="values">Missing</td>
                <td class="values" align="right">5589</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO4IN">LBXTO4IN - Toxoplasma Agglutin interpretation</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO4IN</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma Agglutin interpretation</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma Agglutin interpretation</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">A</td>
                <td class="values">Acute</td>
                <td class="values" align="right">2</td>
                <td class="values" align="right">2</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">C</td>
                <td class="values">Chronic</td>
                <td class="values" align="right">47</td>
                <td class="values" align="right">49</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">E</td>
                <td class="values">Equivocal</td>
                <td class="values" align="right">25</td>
                <td class="values" align="right">74</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">&lt; blank &gt;</td>
                <td class="values">Missing</td>
                <td class="values" align="right">5589</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO5">LBXTO5 - Toxoplasma (Avidity) IgG</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO5</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma (Avidity) IgG</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma (Avidity) IgG</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">0.017 to 0.71</td>
                <td class="values">Range of Values</td>
                <td class="values" align="right">49</td>
                <td class="values" align="right">49</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">.</td>
                <td class="values">Missing</td>
                <td class="values" align="right">5614</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div class="pagebreak">
          <h3 class="vartitle" id="LBXTO5IN">LBXTO5IN - Toxoplasma (Avidity IgG) interpretation</h3>
          <dl>
            <dt>Variable Name: </dt>
            <dd class="info">LBXTO5IN</dd>
            <dt>SAS Label: </dt>
            <dd>Toxoplasma (Avidity IgG) interpretation</dd>
            <dt>English Text: </dt>
            <dd class="info">Toxoplasma (Avidity IgG) interpretation
</dd>
            <dt>Target: </dt>
            <dd> Both males and females 6 YEARS -
			49 YEARS</dd>
          </dl>
          <table class="values">
            <thead>
              <tr>
                <th scope="column" class="values">Code or Value</th>
                <th scope="column" class="values">Value Description</th>
                <th scope="column" class="values">Count</th>
                <th scope="column" class="values">Cumulative</th>
                <th scope="column" class="values">Skip to Item</th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td scope="row" class="values">A</td>
                <td class="values">Acute</td>
                <td class="values" align="right">2</td>
                <td class="values" align="right">2</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">C</td>
                <td class="values">Chronic</td>
                <td class="values" align="right">47</td>
                <td class="values" align="right">49</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">E</td>
                <td class="values">Equivocal</td>
                <td class="values" align="right">0</td>
                <td class="values" align="right">49</td>
                <td class="values">
                </td>
              </tr>
              <tr>
                <td scope="row" class="values">&lt; blank &gt;</td>
                <td class="values">Missing</td>
                <td class="values" align="right">5614</td>
                <td class="values" align="right">5663</td>
                <td class="values">
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </body>
</html>